DOCS-4762 Java Compatibility Requirements Edit (#16718)
DataDog/documentation: content/en/tracing/trace_collection/compatibility/java.md

| Ratpack                 | 1.5+       | Fully Supported | `ratpack`                                      |
| Restlet HTTP Server     | 2.2 - 2.4  | Fully Supported | `restlet-http`.                                |
| Spark Java              | 2.3+       | [Beta][3]       | `sparkjava` (requires `jetty`)                 |
| Spring Boot             | 3          | Fully Supported | `spring-boot`                                  |
| Spring Web (MVC)        | 4.0+       | Fully Supported | `spring-web`                                   |
| Spring WebFlux          | 5.0+       | Fully Supported | `spring-webflux`                               |
| Tomcat                  | 5.5+       | Fully Supported | `tomcat`                                       |
| Vert.x                  | 3.4-3.9.x  | Fully Supported | `vertx`, `vertx-3.4`                           |

**Note**: Many application servers are Servlet compatible and are automatically covered by that instrumentation, such as Websphere, Weblogic, and JBoss.

> Also, frameworks like Spring Boot (version 3) inherently work because it usually uses a supported embedded application server, such as Tomcat, Jetty, or Netty.

Author: ```suggestion
Also, frameworks like Spring Boot (version 3) inherently work because they usually use a supported embedded application server such as Tomcat, Jetty, or Netty.
```

> Also, frameworks like Spring Boot (version 3) inherently work because it usually uses a supported embedded application server, such as Tomcat, Jetty, or Netty).

Author: ```suggestion
Also, frameworks like Spring Boot (version 3) inherently work because it usually uses a supported embedded application server, such as Tomcat, Jetty, or Netty.
```
